pkgs/applications/version-management/redmine/gemset.nix

```diff
@@ -870,9 +881,9 @@
     platforms = [];
     source = {
       remotes = ["https://rubygems.org"];
-      sha256 = "1g0bw2qcl48fxawrdf68l229508z53mrqisavji2lkxzv4w4j2pp";
+      sha256 = "0qzr5j1a1cafv81ib3i51qyl8jnmwdxlqi3kbiraldzpbjh4ln9h";
       type = "gem";
     };
-    version = "0.9.24";
+    version = "0.9.26";
   };
 }
```

Comment:
Missing final new line.

Reply:
Oops... forgot about [that](https://github.com/nix-community/bundix/pull/78). Thanks!